clusterman/batch/node_migration.py

```diff
@@ -53,7 +59,9 @@ def configure_initial(self):
         self.sqs_client = sqs()
         self.migration_workers = {}
         self.migration_configs = {}
+        self.events_in_progress = set()
         self.run_interval = staticconf.read_int("batches.node_migration.run_interval_seconds", 60)
+        self.event_visibilty_timeout = staticconf.read_int("batches.node_migration.event_visibilty_timeout", 15 * 60)
```

Comment:
nit: its nice to have the unit of time in the name of the config option for clarity (e.g, just like the `run_interval_seconds` option above)

Reply:
good catch, I'll PR this fix.